### DISCUSSION TITLE
latest image potentional malicios?

### DISCUSSION THREAD

**Discussion by y0d4a** [Security-Onion-Solutions/securityonion#8792] [Unsupported Versions]
Hi, 

I am sorry for this subject, i hope that is wrong one, but i had very strange situation.
This morning we got report on abuse email that one of our IP is scanning internet for 445 and 1433 port.
After investigation, we found that IP which cause those scans is SO!
![image](https://user-images.githubusercontent.com/3115997/191712792-6b0c3e7b-54fd-443c-ac02-e4cd2b7c0c19.png)
this is one of proof that SO (10.10.46.222) trying brute-force local MSSQL (PAMsystem)..

We isolate SO and now doing some forensic... problem is that this server is downloaded and put in production on Monday (19sept), with, to be honest, not so strong ssh password (but i did not seen brute-force attempt or similar on SO).
I am suspicious that SO is corrupted, but after all what happening on security scene, everything is possible... 

So, i am reporting this to you, just in case if you can take a look at latest image and see is there any suspicious activities on your side.
And, if is possible to get some tips for forensic SO to try to catch what happen....

thank you

**Comment by dougburks** [comment]
We run several instances of the latest image and none of them exhibit this behavior.

You'll want to check to see if somebody logged in with your not so strong ssh password and is performing this activity.

**Comment by y0d4a** [comment]
Sure, we turned off SO and will do detail forensic next week.
I will update when i find what happen.

**Comment by y0d4a** [comment]
Hi, i start to searching on SO image, he is 100% compromised, there is bunch strange connections on him.
Here is from pcap results, can you confirm me which hosts/IPs are legit from SO from the list:

> 
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/srmicv00/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/srmicv00/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



2a01:4f9:3a:4364::3 | easylist-downloads.adblockplus.org
-- | --
109.122.99.212 | karnak.v6.rs
2a01:4f9:4b:1e5f::3 | easylist-downloads.adblockplus.org
2a01:4f9:4b:3e08::3 | easylist-downloads.adblockplus.org
2a01:4f9:6a:1109::3 | easylist-downloads.adblockplus.org
2a01:4f9:6a:1faf::3 | easylist-downloads.adblockplus.org
2a01:4f9:6a:22cf::3 | easylist-downloads.adblockplus.org
2a01:4f9:6b:4def::3 | easylist-downloads.adblockplus.org
2a03:2880:f017:0:face:b00c:0:1 | star.c10r.facebook.com
31.13.87.1 | star.c10r.facebook.com
31.13.87.36 | star-mini.c10r.facebook.com
34.107.221.82 | detectportal.firefox.com
34.117.237.239 | contile.services.mozilla.com
35.227.219.237 | msg.pressplay.cc
40.101.144.98 | HND-efz.ms-acdc.office.com
40.115.154.222 | ejp1-authgw.cloudapp.net
40.126.13.8 | www.tm.a.prd.aadg.akadns.net
40.99.34.2 | HND-efz.ms-acdc.office.com
51.195.115.57 | easylist-downloads.adblockplus.org
51.195.122.232 | easylist-downloads.adblockplus.org
51.255.232.90 | easylist-downloads.adblockplus.org
51.255.236.117 | easylist-downloads.adblockplus.org
51.255.236.119 | easylist-downloads.adblockplus.org
51.255.236.126 | easylist-downloads.adblockplus.org
52.113.194.132 | s-0005.s-msedge.net
52.167.249.196 | settings-prod-eus2-1.eastus2.cloudapp.azure.com
52.182.143.210 | onedscolprdcus10.centralus.cloudapp.azure.com
52.183.220.149 | settings-prod-scus-2.southcentralus.cloudapp.azure.com
52.98.89.2 | HND-efz.ms-acdc.office.com
52.98.89.66 | HND-efz.ms-acdc.office.com
54.216.252.255 | human-tarantula-wy3gj0btg9vmv5qgdfoaab1p.herokudns.com
54.38.153.152 | easylist-downloads.adblockplus.org
54.73.26.109 | human-tarantula-wy3gj0btg9vmv5qgdfoaab1p.herokudns.com
65.108.42.12 | easylist-downloads.adblockplus.org
65.108.42.13 | easylist-downloads.adblockplus.org
65.21.112.201 | easylist-downloads.adblockplus.org
65.21.232.199 | easylist-downloads.adblockplus.org
65.21.236.8 | easylist-downloads.adblockplus.org
65.21.35.75 | easylist-downloads.adblockplus.org
65.21.91.203 | easylist-downloads.adblockplus.org
8.250.163.254 | fg.download.windowsupdate.com.c.footprint.net
8.250.188.126 | fg.download.windowsupdate.com.c.footprint.net
8.252.191.254 | fg.download.windowsupdate.com.c.footprint.net
8.252.73.126 | fg.download.windowsupdate.com.c.footprint.net
8.252.74.126 | fg.download.windowsupdate.com.c.footprint.net
8.8.4.4 | dns.google
8.8.8.8 | dns.google
87.98.243.164 | easylist-downloads.adblockplus.org
87.98.245.61 | easylist-downloads.adblockplus.org
93.184.216.34 | example.org
94.130.143.51 | easylist-downloads.adblockplus.org
95.216.29.90 | easylist-downloads.adblockplus.org
95.216.43.26 | easylist-downloads.adblockplus.org
95.216.73.8 | easylist-downloads.adblockplus.org
95.217.32.92 | easylist-downloads.adblockplus.org
95.217.38.208 | easylist-downloads.adblockplus.org
95.217.43.202 | easylist-downloads.adblockplus.org
95.217.47.15 | easylist-downloads.adblockplus.org
95.217.57.100 | easylist-downloads.adblockplus.org
fe80::8bb9:a200:7f75:1d29 | fe80:0:0:0:8bb9:a200:7f75:1d29%enp0s3.local
fe80::a00:27ff:fee9:b9af | fe80:0:0:0:a00:27ff:fee9:b9af%eth0.local
103.147.22.149 | pool.ntp.org
104.16.229.42 | downloads.boi9osyg1uwtyafn.com
104.16.230.42 | downloads.boi9osyg1uwtyafn.com
104.18.33.94 | www.notion.so
108.128.72.146 | human-tarantula-wy3gj0btg9vmv5qgdfoaab1p.herokudns.com
111.251.202.204 | wendy.dynv6.net
13.107.21.200 | dual-a-0001.a-msedge.net
13.107.21.239 | dual-a-0036.a-msedge.net
13.107.3.128 | s-0001.s-msedge.net
13.107.42.16 | l-0007.l-msedge.net
13.76.217.211 | sa1-authgw.cloudapp.net
135.181.178.148 | easylist-downloads.adblockplus.org
135.181.180.112 | easylist-downloads.adblockplus.org
135.181.181.204 | easylist-downloads.adblockplus.org
135.181.182.211 | easylist-downloads.adblockplus.org
136.243.131.99 | easylist-downloads.adblockplus.org
136.243.40.109 | easylist-downloads.adblockplus.org
136.243.81.48 | easylist-downloads.adblockplus.org
138.201.253.184 | easylist-downloads.adblockplus.org
142.251.42.227 | ssl.gstatic.com
142.251.42.234 | optimizationguide-pa.googleapis.com
142.251.43.10 | optimizationguide-pa.googleapis.com
142.251.43.14 | play.google.com
144.76.109.15 | easylist-downloads.adblockplus.org
148.251.49.239 | easylist-downloads.adblockplus.org
162.159.200.1 | pool.ntp.org
162.159.200.123 | pool.ntp.org
17.253.116.253 | pool.ntp.org
172.217.160.106 | optimizationguide-pa.googleapis.com
172.217.160.74 | optimizationguide-pa.googleapis.com
172.217.163.35 | ssl.gstatic.com
172.217.163.42 | signaler-pa.clients6.google.com
172.217.163.46 | drive.google.com
172.64.154.162 | www.notion.so
176.9.19.104 | easylist-downloads.adblockplus.org
176.9.37.52 | easylist-downloads.adblockplus.org
184.87.235.117 | e8652.dscx.akamaiedge.net
192.0.0.170 | ipv4only.arpa
192.229.232.200 | static-asm.secure.skypeassets.com
195.201.10.74 | easylist-downloads.adblockplus.org
198.244.169.83 | easylist-downloads.adblockplus.org
198.244.173.119 | easylist-downloads.adblockplus.org
20.190.141.32 | www.tm.a.prd.aadg.akadns.net
20.190.141.33 | www.tm.a.prd.aadg.akadns.net
20.190.141.34 | www.tm.a.prd.aadg.akadns.net
20.190.141.35 | www.tm.a.prd.aadg.akadns.net
20.190.141.36 | www.tm.a.prd.aadg.akadns.net
20.190.141.37 | www.tm.a.prd.aadg.akadns.net
20.190.141.38 | www.tm.a.prd.aadg.akadns.net
20.42.73.25 | onedscolprdeus06.eastus.cloudapp.azure.com
2001:41d0:203:55ec::3 | easylist-downloads.adblockplus.org
2001:41d0:203:a57e::3 | easylist-downloads.adblockplus.org
2001:41d0:203:b232::3 | easylist-downloads.adblockplus.org
2001:41d0:203:b47d::3 | easylist-downloads.adblockplus.org
2001:41d0:306:abc::3 | easylist-downloads.adblockplus.org
2001:41d0:700:40dc::3 | easylist-downloads.adblockplus.org
2001:41d0:700:53d9::3 | easylist-downloads.adblockplus.org
2001:41d0:700:5b80::3 | easylist-downloads.adblockplus.org
2001:41d0:800:1f4b::3 | easylist-downloads.adblockplus.org
2001:41d0:800:2be3::3 | easylist-downloads.adblockplus.org
2001:41d0:800:3003::3 | easylist-downloads.adblockplus.org
2001:4860:4860::8844 | dns.google
2001:4860:4860::8888 | dns.google
204.79.197.200 | dual-a-0001.a-msedge.net
204.79.197.239 | dual-a-0036.a-msedge.net
216.58.214.142 | google.com
220.130.51.145 | chat.draytek.com
2404:6800:4012:1::200a | optimizationguide-pa.googleapis.com
2404:6800:4012:2::2003 | ssl.gstatic.com
2404:6800:4012:2::200a | optimizationguide-pa.googleapis.com
2404:6800:4012:2::200e | drive.google.com
2404:6800:4012:3::2003 | ssl.gstatic.com
2404:6800:4012:3::200a | optimizationguide-pa.googleapis.com
2404:6800:4012:4::200e | play.google.com
2404:6800:4012::200a | signaler-pa.clients6.google.com
2600:1901:0:38d7:: | detectportal.firefox.com
2606:2800:147:ff8:129b:22eb:20b:1347 | static-asm.secure.skypeassets.com
2a01:4f8:10a:8a0::3 | easylist-downloads.adblockplus.org
2a01:4f8:10b:1184::3 | easylist-downloads.adblockplus.org
2a01:4f8:13b:dc7::3 | easylist-downloads.adblockplus.org
2a01:4f8:161:3259::3 | easylist-downloads.adblockplus.org
2a01:4f8:161:3287::3 | easylist-downloads.adblockplus.org
2a01:4f8:171:625::3 | easylist-downloads.adblockplus.org
2a01:4f8:173:248f::3 | easylist-downloads.adblockplus.org
2a01:4f8:192:209a::3 | easylist-downloads.adblockplus.org
2a01:4f8:202:4f8::3 | easylist-downloads.adblockplus.org
2a01:4f8:221:1b93::3 | easylist-downloads.adblockplus.org
2a01:4f8:231:491f::3 | easylist-downloads.adblockplus.org
2a01:4f8:252:598e::3 | easylist-downloads.adblockplus.org
2a01:4f9:2a:1bcc::3 | easylist-downloads.adblockplus.org
2a01:4f9:2a:1daa::3 | easylist-downloads.adblockplus.org
2a01:4f9:2a:2945::3 | easylist-downloads.adblockplus.org
2a01:4f9:2a:2c8f::3 | easylist-downloads.adblockplus.org
2a01:4f9:2a:33a3::3 | easylist-downloads.adblockplus.org
2a01:4f9:2b:1cd8::3 | easylist-downloads.adblockplus.org
2a01:4f9:2b:1e1d::3 | easylist-downloads.adblockplus.org
2a01:4f9:2b:1f46::3 | easylist-downloads.adblockplus.org
2a01:4f9:3a:1c98::3 | easylist-downloads.adblockplus.org
2a01:4f9:3a:2085::3 | easylist-downloads.adblockplus.org
2a01:4f9:3a:2855::3 | easylist-downloads.adblockplus.org




</body>

</html>

**Reply by y0d4a** [reply]
also, i research logs and machines which are in the same network where SO was, and still did not found anything suspicious...

**Reply by dougburks** [reply]
Much of that looks like normal web browsing from a web browser with the Adblockplus browser extension. Are you sure that traffic is actually originating from your Security Onion box?

**Reply by Acewiza** [reply]
Don't feel bad.  I often have a hard time understanding what the Onion is showing me here myself, and I been doing this shyt since Moonlight Maze.

**Comment by y0d4a** [comment]
Here are the suspicious hosts which SO contacting, can you confirm me what is from SO what is unknown to you?
Karnak.v6.rs
Server-24.exe-net.net

and can you explain me what kind of UDP packages is this?
![image](https://user-images.githubusercontent.com/3115997/197984125-780244e8-4766-4690-9389-324d76d95a1b.png)



thank you!

**Reply by dougburks** [reply]
Security Onion don't use either of those domain names, so they would be considered unknown.

Your screenshot is very difficult to read and doesn't have much information in it.
